layers/+distribution/spacemacs-base/packages.el

```diff
@@ -815,6 +815,7 @@ below. Anything else exits."
     :config
     (progn
       (projectile-global-mode)
+      (helm-projectile-on)
```

Comment:
Is `helm-projectile` available if the user uses `spacemacs-ivy` layer instead of `spacemacs-helm`? Should probably move this to `helm-projectile`'s configuration in `layers/+completion/spacemacs-helm/packages.el`.